MapboxDirections/MBVisualInstruction.swift

```diff
@@ -104,5 +123,6 @@ open class VisualInstruction: NSObject, NSSecureCoding {
         coder.encode(primaryTextComponents, forKey: "primaryTextComponents")
         coder.encode(secondaryText, forKey: "secondaryText")
         coder.encode(secondaryTextComponents, forKey: "secondaryTextComponents")
+        coder.encode(drivingSide, forKey: "drivingSide")
```

Comment:
Minor: possibly we could considers setting our keys to a constant variable within this `VisualInstruction` class. (Just a future maintenance precaution measure.)

Reply:
Very true, might be solved with https://github.com/mapbox/MapboxDirections.swift/pull/221